### ISSUE TITLE
List of existing PSR-7 compatible HTTP clients

### ISSUE THREAD

**Issue by Nyholm** [php-http/fig-standards#4]
I want to write down a list of HTTP clients that should implement the new PSR. 
- HTTPlug clients and adapters
- [Guzzle6](https://github.com/guzzle/guzzle)
- [Buzz react](https://github.com/clue/php-buzz-react)
- [Windwalker](https://packagist.org/packages/windwalker/http)

**Comment by kelunik** [comment]
Buzz won't be compatible, because it returns a promise, not a response directly.

**Comment by Nyholm** [comment]
True. But it would be a 2 line change to convert's Buzz react's async client to a sync client. That is why I added it on the list.
